server/libs/domain/src/search/search.service.ts

```diff
@@ -155,12 +156,15 @@ export class SearchService {
 
     try {
       // TODO: do this in batches based on searchIndexVersion
-      const assets = this.patchAssets(await this.assetRepository.getAll({ isVisible: true }));
-      this.logger.log(`Indexing ${assets.length} assets`);
+      const assetPagination = usePagination(JOBS_ASSET_PAGINATION_SIZE, (pagination) =>
+        this.assetRepository.getAll(pagination, { isVisible: true }),
+      );
 
-      const chunkSize = 1000;
-      for (let i = 0; i < assets.length; i += chunkSize) {
-        await this.searchRepository.importAssets(assets.slice(i, i + chunkSize), false);
+      for await (const assets of assetPagination) {
+        this.logger.log(`Indexing ${assets.length} assets`);
+
+        const patchedAssets = this.patchAssets(assets);
+        await this.searchRepository.importAssets(patchedAssets, false);
```

Comment:
This message will be logged every 1000 assets, not sure if that's an issue. The `patchAssets` function is empty, but that seems by design so I've kept it

Reply:
In that case, maybe we just change it to a debug log level instead.